File: usth/ICT2.2/labwork/3/C++/shopping-cart-test.cc
#include <iostream>

#include "shopping-cart.h"

using namespace std;

int
main ()
{
  ShoppingCart cart;
  cart.add_to_cart ("foo");
  cart.add_to_cart ("foo");
  cart.add_to_cart ("bar");
  cart.remove_from_cart ("baz");
  cout << cart.count ("foo") << ' '
       << cart.count ("bar") << ' '
       << cart.count ("baz") << endl;
  cart.check_out ();
}

File: usth/ICT2.2/labwork/3/Java/ShoppingCartTest.java
class ShoppingCartTest
{
  public static void main(String... args)
  {
    ShoppingCart cart = new ShoppingCart();
    System.out.println(cart);
    cart.addToCart("foo");
    cart.addToCart("bar");
    cart.addToCart("baz");
    System.out.println(cart);
    cart.removeFromCart("bar");
    System.out.println(cart);
    cart.checkOut();
    System.out.println(cart);
  }
}
